### PR TITLE
Jan 12 IIPC talk slides + save-your-threads > thread-keeper

### PR DESCRIPTION
**Pull request by matteocargnelutti**
- As suggested, I added a very short blog post with my slides from my Jan 12 talk.
- Also renamed the sketch entry for _"Save your threads"_ to _"Thread keeper"_

### CODE REVIEW

File: app/_posts/2023-01-18-thread-keeper-iipc-webinar.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "IIPC Technical Speaker Series: Archiving Twitter"
+author: matteo-cargnelutti
+excerpt_separator: <!--more-->
+---
+I have been invited by the [International Internet Preservation Consortium (IIPC)](https://netpreserve.org/)
```
Comment (bensteinberg) [inline app/_posts/2023-01-18-thread-keeper-iipc-webinar.md]:
```suggestion
I was invited by the [International Internet Preservation Consortium (IIPC)](https://netpreserve.org/) 
```

**Review by bensteinberg — APPROVED** [review]
Looks good -- I made one suggestion, and I'm not sure what to do about the HTML validation failure: 
> Bad value “100%” for attribute “width” on element “iframe”: Expected a digit but saw “%” instead.

File: app/_posts/2023-01-18-thread-keeper-iipc-webinar.md (hunk 1 above)
Comment (matteocargnelutti) [reply app/_posts/2023-01-18-thread-keeper-iipc-webinar.md]: Good catch, thanks :) !

**Comment by matteocargnelutti** [comment]
@bensteinberg I changed strategy, let's see 🥁

**Pull request merged**